runtime/wasmer.go

```diff
@@ -342,6 +355,21 @@ func ext_blake2_256(context unsafe.Pointer, data, length, out int32) {
 	copy(memory[out:out+32], hash[:])
 }
 
+//export ext_blake2_128
```

Comment:
Lets move all these into another file so that this file just defines the runtime struct. A separate package with these in multiple files would be sweeeeeeeet! :surfing_man:

Reply:
i don't think we can do that due to cgo, the ext_ functions and the cgo stuff at the top needs to be there for the NewRuntime to recognize the ext_ functions